src/index.html

```diff
@@ -1,18 +1,62 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="product-card"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="product-card__page">
```

Comment:
The `class` attribute value `product-card__page` does not follow the BEM naming convention. This class appears to be a block-level element, but it is named as an element of the `product-card` block. If this is a block, it should be named with a single name, like `page` or `product-page`.